src/development/variables.md

```diff
@@ -214,6 +226,15 @@ While the same command on the `feature-x` branch would produce:
 }
 ```
 
+### In your application
+
+Check the individual documentation pages for accessing environment variables for your given application language...
+
+* [PHP: the getenv() function](http://php.net/manual/en/function.getenv.php)
+* [Node.js: the process.env object](https://nodejs.org/api/process.html#process_process_env)
+* [Python: the os.environ object](https://docs.python.org/3/library/os.html#os.environ)
+* [Ruby: the ENV accessor](https://ruby-doc.org/core-2.1.4/ENV.html)
```

Comment:
We should do actual code samples here, methinks.  We have the code formatter for it.